### DISCUSSION TITLE
how to declare headings (h1,h2,....) in the new version?

### DISCUSSION THREAD

**Discussion by dimvarvarezos** [globaleaks/globaleaks-whistleblowing-software#4219] [Q&A]
how can somone declare headings (h1,h2,....) in the new version?
in the old version you could decleare them with "#" for example #### for h4.

**Comment by evilaliv3** [comment]
Thank you for your questions @dimvarvarezos 

Also the new version support mark down.

Which part of the application are you trying to customize?

**Reply by dimvarvarezos** [reply]
The "Presentation"

**Comment by evilaliv3** [comment]
@dimvarvarezos , i've actually just retested it and it works as expected.

For example i typed: "# TEST" as presentation

**Reply by dimvarvarezos** [reply]
![Στιγμιότυπο οθόνης 2024-10-09 092502](https://github.com/user-attachments/assets/632345ef-3b9d-401d-8e8f-edf4a4f31ce1)
I tried again but it doesnt recognise "#" as H1 as you can see

**Comment by evilaliv3** [comment]
@dimvarvarezos : the proper markdown syntax requires a space between # and the test like in ```#Whistleblowing Services```

**Reply by dimvarvarezos** [reply]
thank you!!
